io_scene_godot/converters/armature.py

```diff
@@ -88,26 +90,38 @@ def export_bone(pose_bone, self_id, parent_id):
 def attach_bones_to_skeleton(skeleton_node, bone_list):
     """Convert Bone list to attributes of skeleton node"""
     for bone in bone_list:
-        for attr in Bone.attributes:
-            if not getattr(bone, attr) is None:
-                skeleton_node[bone.attr_to_key(attr)] = getattr(bone, attr)
+        bone.id = len(skeleton_node.bone_name_to_id_map)
```

Comment:
It seems to me like this function could quite happily turn into `SkeletonNode.add_bone()`. This would place the job of maintaining consistency of bone_id's on the Skeleton Node rather than the code that adds the bone. (And hides the implementation of the bone_name map)

Having a SkeletonNode as a class is something I'm a little undecided on, but can see the need to maintain the bone_name map for adding the attachments later. I don't think there's any way to avoid it.